docs/reference/commands.md

```
@@ -58,6 +58,7 @@ Compatibility aliases are supported:
 | `codex auth features` | Print implemented feature summary |
 | `codex auth report` | Generate full health report |
 | `codex auth why-selected [--now|--last]` | Explain which account the selector picks now or via the last persisted runtime snapshot |
+| `codex auth rotation enable|disable|status` | Manage the opt-in runtime Responses proxy for live Codex account rotation |
```

markdownlint-cli2 (0.22.1)

[warning] Table column count
Expected: 2; Actual: 4; Too many cells, extra data will be missing

(MD056, table-column-count)